odl/tomo/geometry/geometry.py

```diff
@@ -164,26 +164,6 @@ def det_refpoint(self, mpar):
             The reference point, an `ndim`-dimensional vector
         """
 
-    @abstractmethod
```

Comment:
Scrapped. In the most general case, you don't necessarily have a rotation matrix. For example in PET, the reference point would be stationary.